ivy_tests/test_ivy/test_frontends/test_torch/test_indexing_slicing_joining_mutating_ops.py

```diff
@@ -324,3 +324,54 @@ def test_torch_transpose(
         dim0=dim0,
         dim1=dim1,
     )
+
+
+# movedim
+@given(
+    dtype_and_values=helpers.dtype_and_values(
+        available_dtypes=tuple(
+            set(ivy_np.valid_float_dtypes).intersection(
+                set(ivy_torch.valid_float_dtypes)
+            )
+        ),
+        shape=st.shared(helpers.get_shape(min_num_dims=1), key="shape"),
+    ),
+    source=helpers.get_axis(
+        shape=st.shared(helpers.get_shape(min_num_dims=1), key="shape"),
+    ),
+    destination=helpers.get_axis(
+        shape=st.shared(helpers.get_shape(min_num_dims=1), key="shape"),
+    ),
+    as_variable=st.booleans(),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.torch.movedim"
+    ),
+    native_array=st.booleans(),
+)
+def test_torch_movedim(
+    dtype_and_values,
+    source,
+    destination,
+    as_variable,
+    num_positional_args,
+    native_array,
+    fw,
+):
+    input_dtype, value = dtype_and_values
+    if isinstance(source, int) and not isinstance(destination, int):
```

Comment:
ideally we shouldn't write any logic in the test function, you will have to implement this through some other helper hypothesis function or you can also create your own composite function

Reply:
ok I removed the post-processing. requesting another review, or, how do I merge it myself since you have `approved` the last changes :)